prebid-server/features/pbs-activitycontrols.md

```diff
@@ -190,6 +190,7 @@ Here's the list of the 'potentially restricted activities' that Prebid Server co
 | `transmitEids` | (PBS-Java 2.12 and later) A bid adapter, analytics adapter, or module wants to access and/or transmit extended IDs to their endpoint. | User EIDs are hidden from the adapter or module: `user.eids`, `user.ext.eids` |
 | `transmitPreciseGeo` | A bid adapter, analytics adapter, or module wants to access and/or transmit precise geolocation data to their endpoint | device.geo latitude, longitude, and IP address are rounded off. Specifically, lat and long are truncated to two decimal places, IPv4 masks rightmost 8 bits, IPv6 masks the rightmost bits based on a configured value. In PBS-Java 3.0 and later, device.geo.{country,region,utcoffset} are preserved, but device.geo.{metro,city,zip,accuracy,ipservice,ext} are removed.|
 | `transmitTid` | (PBS-Java only) The source.tid and imp.ext.tid fields can be generated or suppressed by PBS before going to bidders. The ext.prebid.createtid request parameter overrides the allow state of this activity. If `ext.prebid.createtid:false`, transmitTid is "denied". | If "allowed", then source.tid and imp.ext.tid will be generated by PBS. Otherwise, they won't be generated. |
+| `loadExternalScript` | A bid adapter, analytics adapter, or module wants to load external script | Module is not allowed to load external script from `adloader.js` |
```

Comment:
Looks to be on the wrong folder. PBS not JS

Reply:
Should go here instead -> https://docs.prebid.org/dev-docs/activity-controls.html